packages/@aws-cdk/aws-cloudfront/lib/origin.ts

```diff
@@ -194,7 +194,7 @@ export abstract class OriginBase implements IOrigin {
   private renderOriginShield(originShieldRegion?: string): CfnDistribution.OriginShieldProperty | undefined {
     return originShieldRegion
       ? { enabled: true, originShieldRegion }
-      : undefined;
+      : { enabled: false };
```

Comment:
I think I would rather add a new property `originShieldEnabled` which the user can explicitly set to `false` if they want disable it.